docs/UI/Show/index.md

```diff
@@ -22,6 +22,8 @@ Purpose
 |softButtons|[Common.SoftButton](../../common/structs/#softbutton)|false|array: true<br>minsize: 0<br>maxsize: 8|
 |customPresets|String|false|array: true<br>minsize: 0<br>maxsize: 10<br>maxlength: 500|
 |appID|Integer|true||
+|windowID|Integer|false||
```

Comment:
Should the new ShowWidgets.png asset be added to this file? I see the image was created above this file but is not included in the documentation.